### PR TITLE
Bump all of the dependencies to latest.

### PR DESCRIPTION
**Pull request by robertmassaioli**
In this pull request we bump all of the dependencies to get the package to compile on GHC 8.10.3. I was wondering if we could get this package working with as close to latest Haskell as possible. The sooner the better because I want to build my binaries statically!

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -1,8 +1,14 @@
-resolver: nightly-2019-02-17
+resolver: lts-17.0
```
Comment (dzhus) [inline stack.yaml:1]: Any reason to not use the latest available which is https://www.stackage.org/lts-17.9?

Comment (robertmassaioli) [reply stack.yaml]: Done.

**Comment by dzhus** [comment]
Thanks

**Comment by robertmassaioli** [comment]
You're welcome! What's the process for getting it on hackage?

**Comment by dzhus** [comment]
@robertmassaioli it's just been pushed to https://hackage.haskell.org/package/snaplet-redis

**Pull request merged**